frontend/templates/product/sections/ProductOverviewSection/GenuinePartBanner.tsx

```diff
@@ -26,6 +27,7 @@ const partnerCodeToComponentMap: { [key: string]: React.FC } = {
    // Don't have permission yet
    // google_pixel: Google,
    crucial: () => <Crucial px="2" />,
+   hp: Hp,
```

Comment:
```suggestion
   hp: () => <Hp py="2" />,
```

Reply:
Resolved [b282e23](https://github.com/iFixit/react-commerce/pull/1646/commits/b282e230da537bb52ec8a765cab47a3aa6f2c4c1)
Thank you!